web/src/i18n/locales/ja.json

```diff
@@ -64,37 +64,46 @@
     },
     "nominateSpeaker": {
         "bio": {
-            "hint": "被推薦者の簡単な説明",
-            "label": "経歴"
+            "hint": "こ簡潔にご記入ください。",
+            "label": "ご経歴"
         },
-        "city": "都市",
-        "consent": "スピーカーの情報をSpeakHerデータベースに登録する許可をスピーカーから得ています。",
+        "city": "市区町村",
+        "consent": "本推薦者は、スピーカー本人からSpeakHerに推薦・登録する許可を得ています。",
         "email": "Eメール",
         "error": "エラー： {0}",
         "facebook": "FacebookのURL",
         "job": {
-            "title": "職名",
-            "company": "会社"
+            "title": "ご職業",
+            "company": "会社名",
+            "hint": "具体的なご職業（本業）をお書きください。"
         },
         "secondaryAffiliation": {
+          "title": "経歴（その他）",
+          "organization": "所属先（その他）",
+          "hint": "学術研究、ボランティア、副業など、その他の経歴をお書きください。"
         },
         "languages": "言語",
         "linkedIn": "LinkedInのURL",
-        "nameEn": "お名前(英語\/ローマ字)",
-        "nameJa": "お名前（漢字\/かな）",
+        "nameEn": "お名前(英語)",
+        "nameJa": "お名前（漢字）",
```

Comment:
```suggestion
        "nameJa": "お名前（漢字\/かな）",```

Reply:
![name](https://user-images.githubusercontent.com/22512115/107136752-52f13280-6949-11eb-9b56-f0006a4eeea1.png)

Reply:
I wrote down everything on the image.

Reply:
This is interesting feedback and perhaps should be considered in a separate issue. Currently the design is focused on collecting one name for the Japanese version of the site, and one name for the English version of the site. Note how the names switch when changing the language in the toolbar.

For a Japanese user, they would enter Kanji / Kana for their name, and the pronunciation in Romaji
For a non-Japanese user, they could enter their name in Roman characters, then enter the Kana spelling.

We could consider alternate collection methods, but we'll want to keep it simple, and be inclusive of different names. (One of the reasons we don't ask for first or last name is to let people write their full name in the preferred order)